### PR TITLE
Updated the URL syntax in mark down at line 73

### PR DESCRIPTION
**Pull request by ananthulasrikar**
### Describe the proposed changes

Updated the line 73 with URL

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit b356a0b20a437a56adf0b268ed1e8a9234ebe160

https://deploy-preview-680--docker-docs.netlify.com

File: engine/installation/cloud/cloud-ex-aws.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ To get help with your private key file, instance IP address, and how to log into
         $ cd ~/Desktop/keys/amazon_ec2
         $ ssh -i "my-key-pair.pem" ubuntu@xx.xxx.xxx.xxx
 
-    We'll follow the instructions for installing Docker on Ubuntu at https://docs.docker.com/engine/installation/ubuntulinux/. The next few steps reflect those instructions.
+    We'll follow the instructions for installing Docker on Ubuntu [here](/engine/installation/linux/ubuntulinux.md). The next few steps reflect those instructions.
```
Comment (johndmulhausen) [inline engine/installation/cloud/cloud-ex-aws.md]: That's great, but delete "here" and just have the text, "the instructions for installing Docker on Ubuntu" be the link text.

Comment (ananthulasrikar) [inline engine/installation/cloud/cloud-ex-aws.md]: Updated. Thanks so much for correcting me.

**Comment by johndmulhausen** [comment]
Thanks, @ananthulasrikar :)

**Comment by ananthulasrikar** [comment]
Thanks so much !

**Pull request merged**